src/jdk.jdeps/share/classes/com/sun/tools/jdeps/resources/jdeps_de.properties

```diff
@@ -129,4 +131,4 @@ inverse.transitive.dependencies.matching=Inverse transitive Abh\u00E4ngigkeiten,
 internal.api.column.header=JDK-interne API
 public.api.replacement.column.header=Vorgeschlagene Ersetzung
 artifact.not.found=nicht gefunden
-jdeps.wiki.url=https://wiki.openjdk.org/display/JDK8/Java+Dependency+Analysis+Tool
+jdeps.wiki.url=https://wiki.openjdk.java.net/display/JDK8/Java+Dependency+Analysis+Tool
```

Comment:
This seems to change backward. `openjdk.java.net` has changed to `openjdk.org` Same issues are observed in other locales.

Reply:
This and the https->http differences seem to also be a part of the extraction issue. The resource generation files show "http" and "java.net", but I see differently on my local repo that I used for the extraction.

I will run a new extraction job on a fresh repo/branch to see if this issue is reproducible and narrow down the source of the extraction issue.

As discussed with Naoto, these are small edits, and they have been addressed in the latest PR.